Blockzilla/AboutViewController.swift

```diff
@@ -123,7 +123,7 @@ class AboutViewController: UIViewController, UITableViewDataSource, UITableViewD
         case 1:
             url = URL(string: "https://support.mozilla.org/\(AppInfo.config.supportPath)")
         case 2:
-            url = LocalWebServer.sharedInstance.URLForPath("/\(AppInfo.config.rightsFile)")
+            url = Bundle.main.url(forResource: AppInfo.config.rightsFile, withExtension: nil) // TODO LocalWebServer.sharedInstance.URLForPath("/\(AppInfo.config.rightsFile)")
```

Comment:
Do we have an issue we can link with this TODO?

Reply:
Oh good catch! The whole TODO can go away because the `Bundle.main.url(forResource:)` replaces `LocalWebServer.URLForPath`.